README.md

```diff
@@ -102,9 +102,9 @@ conn.execute("select * from dataframe_name")
 Duckdb's SQL parser is based on the PostgreSQL parser, but not all features in PostgreSQL are supported in duckdb. Because the `duckdb_engine` dialect is derived from the `postgresql` dialect, `SQLAlchemy` may try to use PostgreSQL-only features. Below are some caveats to look out for.
 
 ### Auto-incrementing ID columns
-When defining an Integer column as a primary key, `SQLAlchemy` uses the `SERIAL` datatype for PostgreSQL. Duckdb does not yet support this datatype because it's a non-standard PostgreSQL legacy type, so a workaround is to use the `SQLAlchemy.Sequence()` object to auto-increment the key. For more information on sequences, you can find the [`SQLAlchemy Sequence` documentation here](https://docs.sqlalchemy.org/en/14/core/defaults.html#associating-a-sequence-as-the-server-side-default).
+When defining an Integer column as a primary key, `SQLAlchemy` uses the `SERIAL` datatype for PostgreSQL. Duckdb does not yet support this datatype because it's a non-standard PostgreSQL legacy type. In order to fix this, `duckdb-engine` automatically adds a sequence object on model creation to auto-increment the key. For more information on sequences, you can find the [`SQLAlchemy Sequence` documentation here](https://docs.sqlalchemy.org/en/14/core/defaults.html#associating-a-sequence-as-the-server-side-default).
 
-The following example demonstrates how to create an auto-incrementing ID column for a simple table:
+The following example demonstrates how to manually create an auto-incrementing ID column for a simple table:
```

Comment:
I find this a little weird to leave this workaround in the readme, because when I see this, it makes me think that this is a recipe that I might need to implement (but I don't). Can you rephrase this as something like "The following is roughly equivalent to what duckdb-engine does for you automatically"? Basically I want this to be reframed as an explanation, not as a howto.

Reply:
Yeah that was a good rationale. Actually, now that you say that, I think I might prefer removing it entirely as well, instead of adjusting it to an explanation.

Reply:
But @Mause has the final say, I'm just a random user of this project.